Assets/Scripts/UI/HUDController.cs

```diff
@@ -8,13 +8,19 @@ namespace OperationBlackwell.UI {
 	public class HUDController : MonoBehaviour {
 
 		[SerializeField] private Text ActionPointsText_;
+		[SerializeField] private Text WeaponText_;
```

Comment:
Variables should be `camelCase` I know the previous var is not that either so I ask if you could change both of them.